### PR TITLE
Migrate to repository based pipelines

### PR DESCRIPTION
**Pull request by santhosh-thangavel**
1. created a ./builds/latest.yaml and ./builds/release.yaml in the dd-cms repo

#124

### CODE REVIEW

**Review by mikeAdamss — CHANGES_REQUESTED** [review]
So this is going to be a different pattern. The reason is that dd-cms has _2_ applications in it. /plone-5 and /volto.

You're gonna have to make:
./builds/plone-latest.yaml
./builds/plone-release.yaml
./builds/volto-latest.yaml
./builds/volto-release.yaml

the builds will also be more complex (remember the volto tests? that'll need to be in here) so you'll need to look at the steps in the relevant plone and volto cicd terraform for plone and volto and adapt it to yaml.

be careful not to put any full urls in these yaml files please, use the substitution pattern even on the additional steps.

File: builds/volto-latest.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  # Builds Volto-test latest container from the artifact registry and Volto tests are run using container volume
+  - name: 'gcr.io/cloud-builders/docker'
+    dir:
+      - '/workspace/volto/'
+    args:
+      - run
+      - -v
+      - /workspace:/workspace
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:latest
```
Comment (mikeAdamss) [inline builds/volto-latest.yml:29]: so `--destination` is a kaniko argument for when you're building a new image. I think it's ended up here by accident. this should just be the image path, i.e `${_IMAGE_REPOSITORY}/volto-test:latest`

the syntax `>-` is list, you don't need that either.

File: builds/volto-latest.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  # "kaniko" is a build runner whose sole job is to build
+  # container images optimised for use on k8s.
+  # destination is just where on our infrastructure we store
+  # these images.
+  # by default it looks for Dockerfile in the repo root and builds
+  # that though other locations can be specified.
+  # For why we build the image twice with different tags, see
+  # the cicd README.
+  - name: 'gcr.io/kaniko-project/executor:latest'
```
Comment (mikeAdamss) [inline builds/volto-latest.yml:11]: can you add a space and include the comment for this step please

File: builds/volto-latest.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    args:
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:latest
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:$SHORT_SHA
+      - context=dir://volto
+      - Dockerfile=Dockerfile-test
+
+  # Builds Volto-test latest container from the artifact registry and Volto tests are run using container volume
```
Comment (mikeAdamss) [inline builds/volto-latest.yml:20]: in hindsight this is a misleading comment, can you make it "Run volto tests using the volto test container (see above step""

File: builds/volto-latest.yml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      - run
+      - -v
+      - /workspace:/workspace
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:latest
+      - /bin/bash
+      - -c 
+      - cp -R /workspace/volto/src/. /app/src/ && cp /workspace/volto/babel.config.js /app/babel.config.js && yarn test-ci
+
+  - name: 'gcr.io/kaniko-project/executor:latest'
```
Comment (mikeAdamss) [inline builds/volto-latest.yml:34]: Add a comment please, "Build volto image for deployment"

**Review by mikeAdamss — CHANGES_REQUESTED** [review]
Nicely done.

A few minor things only. Can you apply the comments I made on volto-latest to volto-release as well please.

File: builds/volto-latest.yml (hunk 1 above)
Comment (santhosh-thangavel) [reply builds/volto-latest.yml:29]: Removed `--destination` and removed the syntax `>-`

File: builds/volto-latest.yml (hunk 2 above)
Comment (santhosh-thangavel) [reply builds/volto-latest.yml:11]: Added space and the comment

**Comment by santhosh-thangavel** [comment]
> 
Made changes to volto-release as well

**Review by mikeAdamss — APPROVED** [review]
LGTM

**Pull request merged**